### PR TITLE
Question

### PR DESCRIPTION
**Pull request by vikrantkode**

### CODE REVIEW

**Review by krgarima — COMMENTED** [review]
Looks good to me.

File: Pages/Division/division.css [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    min-width: 100px;
+    border-radius: 4px ;
+    cursor: pointer;
+    width: 100%;
+}
+
+a{
+    text-decoration: none;
+    color: var(--text-color);
+}
```
Comment (sharmayatendra) [inline Pages/Division/division.css:48]: target class instead of the element.

File: Pages/Division/division.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    justify-content: flex-start;
+    margin-bottom: 1rem;
+}
+
+.btn{
+    padding: 0.5rem 1rem;
+    margin: 1rem;
+    border:none;
+    font-weight: 600;
+    min-width: 100px;
```
Comment (yashpatel1797) [inline Pages/Division/division.css:39]: use 'rem' here as well.

**Review by yashpatel1797 — APPROVED** [review]
Looks good to merge.

**Review by sharmayatendra — APPROVED** [review]
looks good can make the suggested changes.

**Pull request merged**